spec/database_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
This is a good idea, maybe try to do something similar for the database connection setup process